.github/workflows/promote.yml

```diff
@@ -0,0 +1,42 @@
+name: Promote
+
+on:
+  workflow_dispatch:
```

Comment:
I checked to see if we could put branch restrictions on this, but it seems as though it can be run on any branch where the workflow is defined. This is likely fine as we need these jobs available on all branches (release and master)